themes/src/main/resources/theme/base/admin/resources/partials/user-credentials.html

```diff
@@ -92,9 +92,12 @@
                 <label class="col-md-2 control-label" for="credentialTypeList">{{:: 'disableable-credential-types' | translate}}</label>
 
                 <div class="col-md-6">
-                    <select ui-select2 id="credentialTypeList" ng-model="disableableCredentialTypes" data-placeholder="{{:: 'select-a-type.placeholder' | translate}}" multiple>
-                        <option ng-repeat="credType in user.disableableCredentialTypes" value="{{credType}}">{{credType}}</option>
-                    </select>
+                    <ui-select multiple close-on-select="true" data-ng-model="disableableCredentialTypes" data-ng-change="addDisableableCredentialType(disableableCredentialTypes)" theme="bootstrap" id="credentialTypeList">
+                        <ui-select-match placeholder="{{:: 'select-a-type.placeholder' | translate}}">{{$item}}</ui-select-match>
+                        <ui-select-choices repeat="credType in user.disableableCredentialTypes | filter:$select.search">
+                            <div ng-bind-html="credType | highlight: $select.search"/>
+                        </ui-select-choices>
+                    </ui-select>
```

Comment:
I'm not sure how to properly review this (without hacking in JS console). It seems to me this field is never displayed since introduction of multi-factor auth. Did we just forgot to remove this? @stianst?